jemalloc-sys/src/lib.rs

```diff
@@ -366,8 +366,6 @@ extern "C" {
     ///
     /// This makes the space available for future allocations.
     ///
-    /// If `ptr` is null, no action occurs.
-    ///
     /// # Safety
     ///
     /// The behavior is _undefined_ if:
```

Comment:
We probably need to add to these lists that the behavior is undefined if the pointer is null in both cases.

Reply:
Addressed in [ffdcf7](https://github.com/gnzlbg/jemallocator/pull/144/commits/ffdcf7b62e253868b30ab5ebfed64f16747031de)